README.md

```diff
@@ -1,2 +1,56 @@
 # Greetings
-A site that displays: "Hello World"
+A website that displays the text"Hello World"
+
+![screenshot](images/hello-microverse.gif)
+
+It displays a blue hello world text, centred vertically and horizontally using margin and text-align properties in CSS 
+
+**Built With**
+
+●HTML
+
+●CSS
+
+●Linters
+
+**Live Demo**
+
+[Live Demo Link](https://www.loom.com/share/)16dbc9ea55754157985d307136558780)
```

Comment:
There is no need for this part `16dbc9ea55754157985d307136558780)` kindly remove it